brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1893 Analytics: Added WP background processing support for calculating block count.

### Description
Analytics: Added WP background processing support for calculating block count.

NA

### Types of changes



Breaking change

### How has this been tested?
- Switch to this branch
- Visit Spectra settings page
- Check what data is being sent to analytics

### Checklist:
